test/test_helper.rb

```diff
@@ -19,6 +20,11 @@
 Support::StubCas.stub_requests
 FactoryGirl.find_definitions
 
+VCR.configure do |config|
+  config.cassette_library_dir = "vcr_cassettes"
```

Comment:
I think it might be appropriate for the cassette files to live somewhere inside the test/ directory, what do you think?

Reply:
I didn't notice this blooper, good catch